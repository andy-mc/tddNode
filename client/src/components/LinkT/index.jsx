import React from 'react'

export default function LinkT() {
  return (
    <a href="https://www.tiktok.com/es/">tikTok</a>
  )
}
