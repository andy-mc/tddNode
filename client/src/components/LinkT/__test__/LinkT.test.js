import React from "react";
import renderer from 'react-test-renderer';
import LinkT from '../index';

describe('LinkT', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<LinkT page="https://example.com">Example Site</LinkT>)
      .toJSON();
      expect(tree).toMatchSnapshot(``);
  });
});

