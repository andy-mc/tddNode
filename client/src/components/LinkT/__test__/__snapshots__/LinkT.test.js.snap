// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkT renders correctly 1`] = `
Array [
  <h2>
    HOla
  </h2>,
  <a>
    7
  </a>,
]
`;
